src/types/JobListing.ts

```diff
@@ -0,0 +1,32 @@
+export type Job = {
+  id: number;
+  company: string;
+  logo: string;
+  new: Boolean;
+  featured: Boolean;
+  position: string;
+  role: string;
+  level: string;
+  postedAt: string;
+  contract: string;
+  location: string;
+  languages: string[];
+  tools: string[];
+};
+
+export type JobTransformed = {
+  id: number;
+  company: string;
+  logo: string;
+  new: Boolean;
+  featured: Boolean;
+  position: string;
+  role: string;
+  level: string;
+  postedAt: string;
+  contract: string;
+  location: string;
+  languages: string[];
+  tools: string[];
+  tags: string[];
```

Comment:
Maybe you can create a union type instead of duplicating all the fields.

Reply:
Good call...so something like this... `export type JobTransformed = Job & JobTags;` 

where JobTags is: `type JobTags = {  tags: string[]; };`

Reply:
It seems the following might do the trick...thoughts?

```
export type JobTransformed = Omit<Job, 'new' | 'featured'> & {
  isNew: Boolean;
  isFeatured: Boolean;
  tags: string[];
};
```

Reply:
Created follow up ticket to continue working on this...https://github.com/jkeohan/fe-mentor-job-listings/issues/12